protobuf/flagd/sync/v1/schema.proto

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
Overall usage of `flagd`  terminology.

Given the `sync` proto is not unique to flagd, we can update the comments in this proto definition. 

For example , `Flagd sends this request as the client...` is not true anymore as we have in-process providers capable of obtaining flags over sync protocol. 

So I propose to update the comments :)

Reply:
I agree with this. I think we definitely want to name these `flagd` for now, but it's possible other systems will implement these, especially the sync.proto: it is used by flagd, but also arbitrary sources that flagd can be the client of.

We should try to keep the comments a bit more agnostic of flagd itself, since it's place in this schema is variable, and other components might implement.

I think we should perhaps replace them with `client` / `server` where appropriate?

Reply:
Alright, I have adjusted the comments accordingly